### DISCUSSION TITLE
Has the behavior of the server option changed in useFetch (nuxt 3.1.1)?

### DISCUSSION THREAD

**Discussion by sea19** [nuxt/nuxt#18657] [Questions]
Hello,
I recently discovered that after updating nuxt from 3.0.0 to 3.1.1, changing the server parameter doesn't work at runtime. When the page is refreshed and the server parameter is changed, the request configuration remains the same as before the application was started.

**Example**
```javascript
const server = true;
const url = ref('https://api.nuxtjs.dev/mountains');

const { data } = await useFetch(url, { server });
```
When I change server to false and refresh the page, the request works as if server is true. But the value changes after restarting nuxt. I created a new project to test, but useFetch works the same in it. Is it a bug or is it supposed to be like this?

**Comment by TheAlexLichter** [comment]
I am afraid I don't understand the exact problem. Could you provide a StackBlitz or similar?
